Campus-iOS/Base/Networking/CampusBackend.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  CampusBackend.swift
+//  Campus-iOS
+//
+//  Created by Anton Wyrowski on 29.11.22.
+//
+
+import Foundation
+import GRPC
+import Logging
+
+struct CampusBackend {
+    
+    private static let s = CampusBackend()
+    
+    static let shared = s.client
+    
+    let client: Api_CampusAsyncClient
+    
+    private init() {
+        let group = PlatformSupport.makeEventLoopGroup(loopCount: 1)
+     
+        var logger = Logger(label: "gRPC", factory: StreamLogHandler.standardOutput(label:))
+        logger.logLevel = .debug
+        
+
+        let channel = ClientConnection
+              .usingPlatformAppropriateTLS(for: group)
+              .withBackgroundActivityLogger(logger)
+              .connect(host: "vmott65.in.tum.de", port: 443)
```

Comment:
Magic string -> Also replace with production url at some point

Reply:
Magic string?

Reply:
Well, like magic numbers, you hardcode a string somewhere in the codebase (i know we are doing that a lot in the current TCA). Isn't there a better way to do this? Something like server-side env files